doc/users/next_whats_new/2018_01_25_scale_axis_by_factor.rst

```diff
@@ -0,0 +1,6 @@
+Ability to scale axis by a fixed order of magnitude
+---------------------------------------------------
+
+To scale an axis by a fixed order of magnitude, set the *scilimits* argument 
+of ``Axes.ticklabel_format`` to the same lower and upper limits. Say to scale 
+the y axis by a million (1e6), use ``ax.ticklabel_format(style='sci', scilimits=(6, 6), axis='y')``
```

Comment:
This seems a little mysterious to the un-initiated.  i.e. why the heck do I have to specify 6 twice in this argument.  I'd say something before that about the "old" way that this worked, and how this is a special case. 

I think I agree w/ @tacaswell that this might as well *also* allow a single integer.  Having a plural in the kwarg name isn't really a good reason to not allow this; limits collapse to single values all the time.